docs/content/documentation/content/sass.md

```diff
@@ -20,16 +20,16 @@ structure and base name into the `public` folder:
 ```bash
 .
 └── sass
-    ├── style.scss // -> ./public/style.css
-    ├── indented_style.sass // -> ./public/indented_style.css
+    ├── style.scss // -> /style.css
+    ├── indented_style.sass // -> /indented_style.css
     ├── _include.scss # This file won't get put into the `public` folder, but other files can @import it.
     ├── assets
-    │   ├── fancy.scss // -> ./public/assets/fancy.css
-    │   ├── same_name.scss // -> ./public/assets/same_name.css
+    │   ├── fancy.scss // -> /assets/fancy.css
+    │   ├── same_name.scss // -> /assets/same_name.css
     │   ├── same_name.sass # CONFLICT! This has the same base name as the file above, so Zola will return an error.
     │   └── _common_mixins.scss # This file won't get put into the `public` folder, but other files can @import it.
     └── secret-side-project
-        └── style.scss // -> ./public/secret-side-project/style.css
+        └── style.scss // -> /secret-side-project/style.css
 ```
```

Comment:
This represents the filesystem view, where they should be in a `public` folder

Reply:
Got it.  The docs are a little confusing.  When I see "using sass in zola" I'd assume, that's how I link to the stylesheet to use sass in zola.  I feel like it should be a little more clear.